soc/nordic/validate_base_addresses.c

```diff
@@ -338,7 +338,7 @@ CHECK_DT_REG(cpuflpr_clic, NRF_FLPR_VPRCLIC);
 CHECK_DT_REG(cpuppr_clic, NRF_PPR_VPRCLIC);
 #if defined(CONFIG_SOC_NRF54L15)
 CHECK_DT_REG(cpuflpr_vpr, NRF_VPR00);
-#elif defined(CONFIG_SOC_NRF54H20)
+#elif defined(CONFIG_SOC_NRF54H20) || defined(CONFIG_SOC_NRF9280)
```

Comment:
Couldn't `CONFIG_NRF_PLATFORM_HALTIUM` be used here?

Reply:
Updated.